### PR TITLE
SISRP-10777 - Front End Dev: Credentials Tab - Languages Card - Edit

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-10777

@christianvuerings trivial merge for the controller ~ removed unused property, fix showAdd() param

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/profile/languagesSectionController.js [hunk 1]
```diff
@@ -170,8 +167,11 @@ angular.module('calcentral.controllers').controller('LanguagesSectionController'
   };
 
   var parsePerson = function(data) {
-    var person = data.data.feed.student;
-    var languages = parseLanguages(person.languages);
+    var languages = parseLanguages(_.get(data, 'data.feed.student.languages'));
+
+    languages = _.sortBy(languages, function(o) {
```
Comment (christianvuerings) [inline src/assets/javascripts/angular/controllers/widgets/profile/languagesSectionController.js]: @dfkaye can we rename `o` to `language`?

**Comment by christianvuerings** [comment]
lgtm as soon as the comment is addressed

File: src/assets/javascripts/angular/controllers/widgets/profile/languagesSectionController.js (hunk 1 above)
Comment (dfkaye) [inline src/assets/javascripts/angular/controllers/widgets/profile/languagesSectionController.js]: @christianvuerings sure ~ building now...

**Comment by christianvuerings** [comment]
@dfkaye interesting looks like there might be a merge conflict now.

**Comment by dfkaye** [comment]
@christianvuerings One green travis

**Pull request merged**